deepwell/src/services/view/service.rs

```diff
@@ -174,6 +172,15 @@ impl ViewService {
                 if Self::can_access_page(ctx, user_permissions).await? {
                     debug!("User has page access, return text data");
 
+                    if options.rerender {
```

Comment:
So this is like `/some-page/rerender/true`? I don't think random viewers should be able to do this, only those with edit permission should be able to

Reply:
The viewing permission is stubbed, I'll add another stub for edit permissions